src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -85,18 +85,21 @@ export class AppLauncherGitproviderService implements GitProviderService {
   getGitHubDetails(): Observable<GitHubDetails> {
     return this.getGitHubUserData().pipe(mergeMap(user => {
       if (user && user.login) {
-        let orgs = [];
+        this.gitHubUserLogin = user.login;
+        let orgs: { [name: string]: string } = {};
         return this.getUserOrgs(user.login).pipe(mergeMap(orgsArr => {
           if (orgsArr && orgsArr.length >= 0) {
-            orgs = orgsArr;
-            this.gitHubUserLogin = user.login;
-            orgs.push(this.gitHubUserLogin);
+            this.repositories[''] = AppLauncherGitproviderService.removeOrganizationPrefix(user.repositories);
```

Comment:
Why do we have empty string as a key here?

Reply:
Github doesn't see ones user name as an organisation, but in the select box we have your user and the organisations that the users belongs to and his user-login. So for the user-login we use an empty value, so that we don't send it to the backend.

Reply:
Got it.